bindings/java/src/test/com/apple/foundationdb/test/StackTester.java

```diff
@@ -808,6 +817,17 @@ private static void testTenantTupleNames(Database db) {
 		}
 	}
 
+	private static void testTenantId(Database db) {
```

Comment:
Is this necessary? I originally wrote a unit test during my first few drafts but after adding the relevant parts for `StackOperation`, this might not be needed anymore

Reply:
No, this probably doesn't need to be included anymore. If you want the assertion, you could move it to the processing of the instruction